internal/devconfig/env.go

```diff
@@ -18,7 +18,8 @@ func (c *Config) ComputedEnv(
 	if c.IsEnvsecEnabled() {
 		env, err = envsec.Env(ctx, projectDir)
 		if err != nil {
-			fmt.Fprintf(os.Stderr, "Error reading secrets from envsec: %s\n\n", err)
+			ux.Ferror(os.Stderr, "Error reading secrets from envsec: %s\n\n", err)
```

Comment:
Given that we are continuing on, should this be a `ux.Fwarning` instead?

Reply:
i.e. something like:

```
ux.Fwarning(os.Stderr, "Computing Devbox environment without envsec secrets. There was an error from envsec: %s\n\n", err)
```

Reply:
I'll can switch to warning. My motivation is that it's a fairly big error (not really a warning) so I wanted to make sure it's clear. But generally we don't use errors for commands that end in success.